packages/nube-devtools/src/types.d.ts

```
@@ -0,0 +1,8 @@
+declare global {
+  interface Window {
+    __NUBE_DEVTOOLS_EXTENSION__: boolean
+    nubeSDK: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)